index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
- [ ] Please always ensure you write short, detailed, easy-to-understand descriptions of pull requests. 
    Kindly edit the pull request with a proper description. 
     Instead of writing "pull request" as your description, you write something like: 

     ## I implemented the following: 

    - Added 3 files (each one in a separate commit):
         - Added index.html file that includes a header with the text "Hello Microverse!". I used `<h1>` tag for that.
         - Added styles.css file that includes one class with styles for the `<h1>`.
         - A test.md file with the text .
    - Added `.gitignore` file with node_module folder and test.md file in it.

With this, it is easy for anyone to have an idea of what you implemented.

Reply:
- [ ] [REQUIRED] Please edit also your Pull Request summary because like your PR title the summary should describe introduced changes in more detail. +1
You can check the example below :arrow_down:
![184951982-6f947437-2d41-415c-b6d7-b26ae0855ea3](https://user-images.githubusercontent.com/6615394/185222492-a9babd67-052f-4fff-a923-4122bad15ab0.png)

Please replace your PR summary description

![Screenshot from 2022-08-17 19-04-37](https://user-images.githubusercontent.com/6615394/185222654-c99d210a-c015-4f06-b672-3af540c28df8.png)